homeassistant/components/starline/services.yaml

```diff
@@ -0,0 +1,10 @@
+update_state:
```

Comment:
We already have a generic update entity service that can be used for this .

Reply:
But this service updates all entities in integration..